docker/mu-plugins/conflicting-plugin.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * Plugin Name: Conflicting Plugin
+ * License: GPL2+
+ */
+error_log( 'loading conflicting plugin' );
+/**
+ * THIS CODE WOULD NEED TO BE DUPLICATED IN EACH PLUGIN...
+ */
+if ( ! function_exists( 'jetpack_enqueue_library' ) ) {
+	function jetpack_enqueue_library( $class_name, $version, $path ) {
+		global $jetpack_libraries;
+		if ( ! is_array( $jetpack_libraries ) ) {
+			$jetpack_libraries = array();
+		}
+		if ( ! isset( $jetpack_libraries[ $class_name ] ) ) {
+			$jetpack_libraries[ $class_name ] = array();
+		}
+		$jetpack_libraries[ $class_name ][] = array( 'version' => $version, 'path' => $path );
+	}
+
+	// add the autoloader
+	spl_autoload_register( function ($class_name) {
+		global $jetpack_libraries;
+		if ( ! isset( $jetpack_libraries[ $class_name ] ) ) {
```

Comment:
As per comment above, this whole function could just be:

```php
global $jetpack_libraries;
if ( isset( $jetpack_libraries[ $class_name ] ) ) { 
  if ( ! did_action( 'plugins_loaded' ) ) { 
    // do warning 
  }
  require_once $jetpack_libraries[ $class_name ]['path'];
}
```

Note that in this example, I nested the `plugins_loaded` check inside the test which determines if our class is registered, since we don't want to perform that check for ANY missing class (some may be from other plugins, etc, and loading those before plugins_loaded may be totally fine)